### DISCUSSION TITLE
I debugged the Karaoke mode

### DISCUSSION THREAD

**Discussion by gjnave** [ggml-org/whisper.cpp#2348] [Ideas]
\Here's what I discovered:
1) it requires a font that is not native to WSL or to Windows
a) i used this in Windows: main.exe -m .\models\ggml-base.en.bin -f .\samples\jfk.wav -owts -fp C:\Windows\Fonts\cour.ttf 
b) it does not work in WSL and I could not get a proper font
2) once the .wts is created, it must be ran in WSL (i could not get it formatted to work as a bat)
3) when it is executed, it errors for this reason:
a) it cannot find ./samples/jfk.mp4 
a1) the reason is that (evidently) ffmpeg does not like the ./ convention. 
4) to fix it, I went through the .wts file and remove .\samples from 3 instances and left it as    \jfk.wav  or \jfk.mp4
a) this fixed it and the mp4 was created.
b) however the fonts didnt match

In summary, you need to fix how the .wts is created.
Thanks for this project!
